fmcg_store/data/product_attribute.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="product_attribute_1" model="product.attribute">
+        <field name="name">Grammage</field>
+    </record>
+    <record id="product_attribute_5" model="product.attribute">
+        <field name="name">Pack of size</field>
+        <field name="display_type">select</field>
+    </record>
```

Comment:
I don't see where those attributes are used, in which flow we check this?

Reply:
So this attributes are set in product.template (product_template_31,product_template_34,)

Reply:
Ok, but then just create several articles with the different sizes of the packs (2 sizes of packs per product should be enough)

Reply:
Or (maybe better), use the product packaging option from stock